_rules/aria-required-owned-element-bc4a75.md

```diff
@@ -0,0 +1,269 @@
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This element with the `list` role only owns elements with the `listitem` role. The `listitem` role is one of the [required owned elements][] for `list`.
+
+```html
+<div role="list">
+	<span role="listitem">Item 1</span>
+	<span role="listitem">Item 2</span>
+</div>
+```
+
+#### Passed Example 2
+
+This element with the `tablist` role only owns elements with the `tab` role. The `tab` role is one of the [required owned elements][] for `tablist`.
+
+```html
+<div role="tablist">
+	<span role="tab">Tab 1</span>
+	<span role="tab">Tab 2</span>
+</div>
+```
+
+#### Passed Example 3
+
+This element with the `grid` role only owns elements with the `row` role, and the element with the `row` role only owns elements with the `cell` role. The `row` role is one of the [required owned elements][] for `grid`, and `cell` is one of the [required owned elements][] for `row`.
+
+```html
+<table role="grid">
+	<tr role="row">
+		<span role="cell">Item 1</span>
+	</tr>
+</table>
+```
+
+#### Passed Example 4
+
+This element with the `menu` role only owns elements with the `menuitem`, `menuitemradio` and `menuitemcheckbox` role. These roles are one of the [required owned elements][] for `menu`. The element with the `none` role is not [owned by][] the `menu` because it is not [included in the accessibility tree][].
+
+```html
+<ul role="menu">
+	<li role="none"></li>
+	<li role="menuitem">Item 1</li>
+	<li role="menuitemradio">Item 2</li>
+	<div role="menuitemcheckbox">Item 3</div>
+</ul>
+```
+
+#### Passed Example 5
+
+This element with the `list` role only owns elements with an implicit `listitem` role. The `listitem` role is one of the [required owned elements][] for `list`.
+
+```html
+<div role="list">
+	<li>Item 1</li>
+</div>
+```
+
+#### Passed Example 6
+
+This element with the `list` role only owns elements with the `listitem` role through the `aria-owns` attribute. The `listitem` role is one of the [required owned elements][] for `list`.
+
+**Note:** This test case follows the definition of [owned by][]. If implemented differently, this definition could cause differences in outcome of this test case.
+
+```html
+<div role="list" aria-owns="id1"></div>
+<div id="id1" role="listitem">Item 1</div>
+```
+
+#### Passed Example 7
+
+This element with the `list` role only owns elements with the `listitem` role, or elements with the `group` role, in which each element has the `listitem` role. The `listitem` role on its own, as well as in a `group` are in the [required owned elements][] for `list`.
+
+```html
+<div role="list">
+	<span role="listitem">Item 1</span>
+	<div role="group">
+		<span role="listitem">Item 2</span>
+		<span role="listitem">Item 3</span>
+	</div>
+</div>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This element with the `list` role owns elements without the `listitem` role. The `listitem` role is one of the [required owned elements][] for `list`.
```

Comment:
```suggestion
This element with the `list` role does not owns an element without any of its [required owned elements][] (`listitem` or `group` containing `listitem`).
```
Because 
```html
<div role="list">
  <div role="group">
    <li>Foo</li>
  </div>
</div>
```
is matching the description (`list` owns elements without `listitem` role) but passes the rule.